app/furniture/marketplace/checkouts_controller.rb

```diff
@@ -36,20 +35,8 @@ def create
       end
     end
 
-    helper_method def shopper
-      @shopper ||= if current_person.is_a?(Guest)
-        Shopper.find_or_create_by(id: session[:guest_shopper_id] ||= SecureRandom.uuid)
-      else
-        Shopper.find_or_create_by(person: current_person)
-      end
-    end
```

Comment:
I pulled these into `marketplace/controller.rb` since it was being used in many places.